tests/dynamic_friction.prm

```diff
@@ -47,20 +35,26 @@ subsection Mesh deformation
 end
 
 subsection Material model
-
-  set Model name = dynamic friction
-  subsection Dynamic Friction
+  set Model name = visco plastic
+  subsection Visco Plastic
     set Densities         = 2800
     set Viscosity averaging scheme = harmonic
-    subsection Viscosities
-       set Minimum viscosity = 1e19
-       set Maximum viscosity = 1e25
-       set Reference strain rate = 1e-15
-       set Background Viscosities = 1.e25, 1.e20
-       set Coefficients of static friction = 0.5547 , 0.5
-       set Coefficients of dynamic friction = 0.6 , 0.5
-       set Cohesions = 20e6
-    end
+    set Minimum viscosity = 1e19
+    set Maximum viscosity = 1e25
+    set Reference strain rate = 1e-15
+
+    # Dislocation creep parameters for
+    set Prefactors for dislocation creep          =      1e-20, 1e-25
```

Comment:
Although the background viscosities were not used in the old test, there is a factor 2 in the creep law that you should take into account in the prefactor to get the values of 1e19 and 1e25.

Reply:
ok. What exactly would I have to change here? 
And do I really have to? The viscosities agree with the old test result when I apply the changes I describe below

Reply:
* the old material model did not consider the background viscosities, except for the reference viscosity, which is set to the first background viscosity
* the old test specified background viscosities
* the new test specifies background viscosities
* the new material model does use background viscosities
* the reference viscosity in the new material model should be set from the input file (although this is no longer used in the scaling of the Stokes equations)
* the new material model uses eta_dislocation = 1/2 * 1/A * strainrate ^(1-n)/n * exp((E+VP)/(nRT)), which means with n=1, V=0, E=0 that 1e20 = 1/2 * 1/A -> A = 5e-21 and similar A = 5e-26
* So just change the prefactors and set the reference viscosity to 1e20 for completeness.